test/e2e/linuxbridge_test.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	yaml "github.com/ghodss/yaml"
+
+	apis "github.com/nmstate/kubernetes-nmstate/pkg/apis"
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 60
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+	createBridgeCr       = "deploy/crds/create-br1-linux-bridge.yaml"
+	deleteBridgeCr       = "deploy/crds/delete-br1-linux-bridge.yaml"
+	firstNodeName        = "node01" // TODO: Get it from cluster
+	bridgeName           = "br1"
+)
+
+func TestLinuxBridge(t *testing.T) {
+
+	nodeNetworkStateList := &nmstatev1.NodeNetworkStateList{}
+	err := framework.AddToFrameworkScheme(apis.AddToScheme, nodeNetworkStateList)
+	if err != nil {
+		t.Fatalf("failed to add custom resource scheme to framework: %v", err)
+	}
+
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	err = ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	if err != nil {
+		t.Fatalf("failed to initialize cluster resources: %v", err)
+	}
+
+	// get namespace
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// get global framework variables
+	f := framework.Global
+	// wait for memcached-operator to be ready
+	err = e2eutil.WaitForOperatorDeployment(t, f.KubeClient, namespace, "memcached-operator", 1, time.Second*5, time.Second*30)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	manifest, err := ioutil.ReadFile(createBridgeCr)
+	if err != nil {
+		t.Fatal(err)
+	}
+	state := nmstatev1.NodeNetworkState{}
+	err = yaml.Unmarshal(manifest, &state)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = f.Client.Create(context.TODO(), &state, &framework.CleanupOptions{
+		TestContext:   ctx,
+		Timeout:       cleanupTimeout,
+		RetryInterval: cleanupRetryInterval})
+	if err != nil {
+		t.Fatal(err)
+	}
+	ctx.AddCleanupFn(func() error {
```

Comment:
mind adding an empty line before this one?

Reply:
Done